### PR TITLE
Simple screen with SDL

### PR DESCRIPTION
**Pull request by alexandrefresnais**

### CODE REVIEW

File: guest/splash.h [hunk 1]
```diff
@@ -0,0 +1,716 @@
+#ifndef SPLASH_HEADER
+#define SPLASH_HEADER
+
+#define SPLASH_WIDTH 91
+#define SPLASH_HEIGHT 93
+
+unsigned char splash[SPLASH_WIDTH * SPLASH_HEIGHT] = {
```
Comment (18AX) [inline guest/splash.h]: I think this should be moved to the guest32.c

File: include/blackhv/framebuffer.h [hunk 2]
```diff
@@ -0,0 +1,11 @@
+#ifndef FRAMEBUFFER_HEADER
+#define FRAMEBUFFER_HEADER
+
+#include <blackhv/types.h>
```
Comment (18AX) [inline include/blackhv/framebuffer.h]: These includes are not needed here

File: include/blackhv/screen.h [hunk 3]
```diff
@@ -0,0 +1,11 @@
+#ifndef SCREEN_HEADER
+#define SCREEN_HEADER
+
+typedef struct vm vm_t;
+typedef struct screen screen_t;
+
+s64 screen_init(vm_t *vm, u64 framebuffer_phys);
```
Comment (18AX) [inline include/blackhv/screen.h:9]: You are using u64 without including `<blackhv/types.h>`

File: guest/splash.h (hunk 1 above)
Comment (alexandrefresnais) [reply guest/splash.h]: Done in 3ed1f22117bee7b2c93ef9f988519be5dbc28fa6

File: include/blackhv/framebuffer.h (hunk 2 above)
Comment (alexandrefresnais) [reply include/blackhv/framebuffer.h]: Done in f2a54e1a1774527adae98bdbecfddd344753066a

File: include/blackhv/screen.h (hunk 3 above)
Comment (alexandrefresnais) [reply include/blackhv/screen.h:9]: Done in 28edf93ff9f10a52b850beb9aac20359f15f220c

**Review by 18AX — APPROVED** [review]
LGTM

**Pull request merged**